b2sdk/raw_api.py

```diff
@@ -949,6 +959,8 @@ def upload_file(
         :param legal_hold: legal hold setting for the file
         :param custom_upload_timestamp: custom upload timestamp for the file
         :param cache_control: an optional cache control setting. Syntax based on the section 14.9 of RFC 2616. Example string value: 'public, max-age=86400, s-maxage=3600, no-transform'.
+        :param expect_100_continue: whether to use 'Expect: 100-continue' header
+        :param expect_100_timeout: timeout of 100 response when expect_100_continue is True
```

Comment:
sorry for another change here, but now that I look at it - wouldn't `expect_100_timeout` being set to 0 be basically the same think and we could skip adding `expect_100_continue` param altogether?